### PR TITLE
Ready if the last block is less than 1 minute old

### PR DESCRIPTION
**Pull request by ouziel-slama**
- [x] API v1
- [x] API v2

### CODE REVIEW

File: counterparty-core/counterpartycore/lib/api/util.py [hunk 1]
```diff
@@ -11,17 +12,20 @@
 logger = logging.getLogger(config.LOGGER_NAME)
 
 
-def check_last_parsed_block(blockcount):
+def check_last_parsed_block(db, blockcount):
     """Checks database to see if is caught up with backend."""
+    last_block = ledger.get_last_block(db)
```
Comment (adamkrellenstein) [inline counterparty-core/counterpartycore/lib/api/util.py:17]: two minutes or one minute? Am I reading this right?

Comment (adamkrellenstein) [reply counterparty-core/counterpartycore/lib/api/util.py:17]: (two minutes seems like a lot!)

File: release-notes/release-notes-v10.1.2.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ To easily migrate to the new API, an equivalence table is available in the docum
 
 ## Codebase
 * New ReST API
+* APIs return ready if the last block is less than 2 minutes old
```
Comment (ouziel-slama) [inline release-notes/release-notes-v10.1.2.md:23]: oh got it lol! fixing that!

File: counterparty-core/counterpartycore/lib/api/util.py (hunk 1 above)
Comment (ouziel-slama) [reply counterparty-core/counterpartycore/lib/api/util.py:17]: fixed the title and the release notes.

**Pull request merged**